core/src/util/yamlHelper.cpp

```diff
@@ -64,30 +64,44 @@ glm::vec4 getColorAsVec4(const Node& node) {
 }
 
 std::string parseSequence(const Node& node) {
+    if (node.IsSequence()) { return ""; }
+
     std::stringstream sstream;
     for (const auto& val : node) {
-        try {
-            sstream << val.as<double>() << ",";
-        } catch (const BadConversion& e) {
-            try {
-                sstream << val.as<std::string>() << ",";
-            } catch (const BadConversion& e) {
-                LOGE("Float or Unit expected for styleParam sequence value");
-            }
+        if (!val.IsScalar()) {
+            // LOG
+            return "";
+        }
+
+        double value;
+        if (getDouble(val, value)) {
+            sstream << value << ",";
+        } else {
+            sstream << val.as<std::string>() << ",";
         }
     }
     return sstream.str();
 }
 
-bool getDouble(const Node& node, double& value, const char* name) {
-    try {
-        value = node.as<double>();
-        return true;
-    } catch (const BadConversion& e) {}
+bool getDouble(const Node& node, double& value) {
 
-    if (name) {
-        LOGW("Expected a floating point value for '%s' property.:\n%s\n", name, Dump(node).c_str());
+    if (node.IsScalar()) {
+        const std::string& s = node.Scalar().c_str();
```

Comment:
sure, that's what I intended - learned one more weirdness of c++ today. Wouldn't have expected the above code to work :)

Some 'rationale'  https://herbsutter.com/2008/01/01/gotw-88-a-candidate-for-the-most-important-const/